tracer/src/Datadog.Trace/AppSec/Waf/Context.cs

```diff
@@ -36,6 +36,11 @@ public Context(IntPtr contextHandle, WafNative wafNative, Encoder encoder)
 
         public IResult Run(ulong timeoutMicroSeconds)
         {
+            if (disposed)
+            {
+                throw new Exception("Can't run WAF when disposed");
```

Comment:
Nit: Use the `ThrowHelper`

Reply:
Fixed in f9cd451ce.

Reply:
Done in 1eb7fb6c959a504cb9ea4d6119effd24b94aeb4b